### ISSUE TITLE
Update quick start docs

### ISSUE THREAD

**Issue by ubershmekel** [theiviaxx/Frog#195]
I tried using http://frog.readthedocs.io/en/latest/admin/quickstart.html

Here are issues I've experienced trying to set up the new frog:

* `'django.contrib.comments',` -> `'django_comments` in INSTALLED_APPS
* `FROG_FFPROBE` isn't mentioned in the example settings.py
* `./manage.py syncdb` -> `./manage.py makemigrations` (then `migrate`)
* Deprecation warnings:

`Support for string view arguments to url() is deprecated and will be removed in Django 1.10`
`RemovedInDjango110Warning: You haven't defined a TEMPLATES setting. You must do so before upgrading to Django 1.10.`
`RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead`

Now I'm wrestling with http 405 responses.

**Comment by ubershmekel** [comment]
@theiviaxx do you have an idea on how to get past the 450 status?

**Comment by theiviaxx** [comment]
Yes, so basically frog was split into front and back ends.  This repo is the backend and the root url '/frog/' is not accessible via GET request.  I'm in the process of getting a quickstart script up and running that should be available for testing tonight.  This will give you a functional back end, the built front end files and an nginx conf file to use to serve it all up.

Im also making a docker image that should be completely self contained.

Once these are done i'll update the docs to reflect the new setup process
